nixos/modules/programs/localsend.nix

```diff
@@ -12,14 +12,19 @@ in
   options.programs.localsend = {
     enable = lib.mkEnableOption "localsend, an open source cross-platform alternative to AirDrop";
 
-    openFirewall = lib.mkEnableOption "opening the firewall port ${toString firewallPort} for receiving files" // {
-      default = true;
-    };
+    package = lib.mkPackageOption pkgs "localsend" { };
```

Comment:
What's the point?

> **Note**: You shouldn’t necessarily make package options for all of your modules. You can always overwrite a specific package throughout nixpkgs by using [nixpkgs overlays](https://nixos.org/manual/nixpkgs/stable/#chap-overlays).

https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/development/option-declarations.section.md#mkpackageoption-sec-option-declarations-util-mkpackageoption

Reply:
Honestly, I don't think that note in the documentation is helpful. There are no clear guidelines as to when we should or shouldn't have a `package` option. I have had numerous occasions, where I would have preferred to have a `package` option, as overlays are always more work to write and maintain.